frontend/web/components/pages/ChangeRequestPage.js

```diff
@@ -27,7 +27,7 @@ const ChangeRequestsPage = class extends Component {
   static contextTypes = {
     router: propTypes.object.isRequired,
   }
-
+  // TODO: review this function users?.filter((v) => approvals?.includes(v.group))
```

Comment:
Does this still need reviewing?

Reply:
This is not related to this PR, I think it is related to this issue https://github.com/Flagsmith/flagsmith/issues/2823,
Comment deleted.